index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <ul>
+        <li><img src="My Logo.svg" alt="Logo svg" /></li>
+        <li><img src="Union.png" alt="Menu Button" /></li>
+      </ul>
+    </header>
+    <div class="headline">
```

Comment:
- [ ] I think it would be better to wrap the headline section with `<section>` semantic tag instead of `<div>` tag 👌
> Fo Example 💡
```
<section class="headline">
 ...
</section>
```